### PR TITLE
Involve non-trainable policies into current workflow

### PR DESCRIPTION
**Pull request by lihuoran**
# Description

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [ ] New feature
- [ ] Test
- [ ] Doc update
- [ ] Docker update

## Related Component

- [ ] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [ ] Linux
- Python version:
  - [ ] 3.6
  - [ ] 3.7
- Key information snapshot(s):

## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [ ] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: examples/rl/main_inference_only.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
```
Comment (Jinyu-W) [inline examples/rl/main_inference_only.py:2]: Would it let me feels like this script is also for learnt_policy

**Review by Jinyu-W — CHANGES_REQUESTED** [review]

File: examples/rl/supply_chain/algorithms/ppo.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+def get_policy(state_dim: int, action_num: int, name: str) -> DiscretePolicyGradient:
+    return DiscretePolicyGradient(name=name, policy_net=MyActorNet(state_dim, action_num))
+
+
+def get_ppo(state_dim: int, name: str) -> DiscretePPOTrainer:
+    return DiscretePPOTrainer(
+        name=name,
+        params=DiscretePPOParams(
+            device="cpu",
```
Comment (Jinyu-W) [inline examples/rl/supply_chain/algorithms/ppo.py:122]: set as a config?

File: examples/rl/supply_chain/env_sampler.py [hunk 3]
```diff
@@ -381,7 +384,15 @@ def _update_global_features(self, state) -> None:
         state["global_time"] = self._learn_env.tick
 
     def _post_step(self, cache_element: CacheElement, reward: Dict[Any, float]) -> None:
-        pass
+        tick = cache_element.tick
```
Comment (Jinyu-W) [inline examples/rl/supply_chain/env_sampler.py:387]: Could also read the `balance` as the metric

File: maro/rl/rollout/env_sampler.py [hunk 4]
```diff
@@ -326,8 +327,8 @@ def sample(self, policy_state: Optional[Dict[str, object]] = None, num_steps: Op
                 )
             )
             # Update env and get new states (global & agent)
-            _, self._event, done = self._env.step(list(env_action_dict.values()))
-            self._state, self._agent_state_dict = (None, {}) if done \
+            _, self._event, terminal = self._env.step(list(env_action_dict.values()))
```
[the remark this answers is not in the document]
Comment (Jinyu-W) [reply maro/rl/rollout/env_sampler.py:330]: use `done` / `is_done` here
